### ISSUE TITLE
Non-conforming contrast on top skip link

### ISSUE THREAD

**Issue by CorgiDev** [CorgiDev/CorgiDevSite#31]
**Issue:** The color contrast on the top skip link is below the WCAG AA threshold.

**Additional Details:** Per the Accessibility Insights automated check:

> Element has insufficient color contrast of 3.62 (foreground color: #ffffff, background color: #dd610e, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

Issue could be resolved by changing the color of the link or increasing the font size. See [WCAG Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) for additional details.
